src/plugins/ping.js

```diff
@@ -18,7 +17,7 @@ const register = (server) => {
         const url = request.query.url;
 
         try {
-          await got.get(url);
+          await got.head(url);
```

Comment:
Nitpick: I assume that `head` is what should happen here, instead of a complete GET, but maybe we should make that change in a separate commit/PR? That way we could (for example) revert the memoization changes later if needed, without also reverting the GET-to-HEAD changes.